### ISSUE TITLE
doesnt work on 6800xt

### ISSUE THREAD

**Issue by ssateneth** [hoffr/RadeonClockEnforcer#6]
tool doesnt work because overdriventool doesnt support radeon 6k series. any fix?

**Comment by hoffor** [comment]
Not my perogative unfortunately. Does your 6000 card still have the issues RadeonClockEnforcer aims to solve?

**Comment by ssateneth** [comment]
Yeah, the clocks dip down a lot during unigine heaven. Max is about 2380 core, but it will dip down to 1500, 1200, or less at times.

but apparently it can be managed by using built in overclock management in drivers because there is a "minimum" core clock that can be set but it cant be set the same as the max core clock, it has to be at least 100 mhz apart. It's something though so I will probably use that
